src/com/theironyard/novauc/User.java

```diff
@@ -0,0 +1,101 @@
+package com.theironyard.novauc;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Created by peter on 2/14/17.
+ */
+public class User {
+
+
+
+    public static ArrayList<InventoryItem> tracker = new ArrayList<>();
+    public static Scanner input = new Scanner(System.in);
+    public static Scanner inputNumbers = new Scanner(System.in);
+
+    public String nameUser;
+    int quantityUser;
+    String categoryUser;
+    int loopdioop = 1;
+    String name;
+    int quantity;
+
+    public void menu() {
+        while (loopdioop == 1) {
+            //ToDo: fix issue of looping extra time
+            System.out.println("What would you like to do? add / list / exit");
+            switch (input.nextLine()) {
+
```

Comment:
Peter again really nice project design.  Your understand of how to accomplish the requirements while breaking your code into logical methods is very impressive.  Keep up the good work.